native/membrane_videocompositor/src/new_compositor.rs

```diff
@@ -0,0 +1,62 @@
+use std::sync::{Arc, Mutex};
```

Comment:
`new_compositor.rs` is a little iffy for be, but I understand it's going to change soon

Reply:
yes, but I don't know what to rename it to.